src/http/message.h

```diff
@@ -0,0 +1,39 @@
+#ifndef MESSAGE_H
+#define MESSAGE_H
+
+#include <string>
+#include <vector>
+
+/*
+ * Interface for HTTP messages. Used to hold common code
+ * for HTTP Requests and Responses
+ */
+class Message {
+protected:
+	// Effectively namespaced constant for carriage returns
+
+	std::vector<std::string> headers_;
```

Comment:
I haven't been using trailing underscores for member variables. Do you want to from now on?